substitute_values.sh

```diff
@@ -26,7 +26,9 @@ function deployment_substitution() {
 
   for f in ${DIRECTORY}/*
   do
-    sed --expression="s#${KEY}#${VALUE}#" --in-place $f
+    newf=`mktemp`
+    sed -e "s#${KEY}#${VALUE}#"  < $f > ${newf}
```

Comment:
Looking at this, I realize we are engaged in a few bad practices that I don't want to propagate.

Please change to:
```bash
newf=$(mktemp)
sed -e "s#${KEY}#${VALUE}#"  < "${f}" > "${newf}"
mv -f "${newf}" "${f}"
```